### DISCUSSION TITLE
[Feature] Explain buffer settings

### DISCUSSION THREAD

**Discussion by ann0see** [jamulussoftware/jamulus#1136] [Documentation]
**Is your feature request related to a problem? Please describe.**

Users coming through the installation guides don't get told know how to handle buffer settings

**Describe the solution you'd like**

Describe buffer settings on the onboarding page

**Comment by gilgongo** [comment]
Isn't this more for the installation guides? In the [Linux guide](https://jamulus.io/wiki/Installation-for-Linux) we mention setting to 128 at first and then only changing if you get issues. For Win/Mac it'll depend on the driver though I guess. But either way, mucking about with buffers shouldn't be necessary beyond initial setup really.

**Comment by dcorson-ticino-com** [comment]
I have seen no disadvantages in setting to automatic and leaving that setting.
Sometimes when connected to servers with flaky connections the buffering gets out of hand, but that is easily seen in the overall delay times and choosing a server with a better connection resolves things again.
I had even considered if it would be better to default to automatic (which I think is the case) and remove the buffer sliders from the setup page, but make a button "Manual Buffering" or similar, opening a small window with just the buffers for those who really want to play with them.

**Comment by gilgongo** [comment]
Ah, by "buffer settings" are we talking about jitter buffers, audio card buffers, or both? I would imagine jitter buffers would definitely not be something we'd want to bother new users with.

**Comment by dcorson-ticino-com** [comment]
Oh, yes I was talking about jitter buffer.
The interface buffer, set once and forgotten.
